pkg/udev/common_test.go

```diff
@@ -141,3 +143,28 @@ func TestGetUid(t *testing.T) {
 	expectedUid := NDMPrefix + util.Hash(diskDetails.Wwn+diskDetails.Model+diskDetails.Serial+diskDetails.Vendor)
 	assert.Equal(t, expectedUid, device.GetUid())
 }
+
+func TestGetDevLinks(t *testing.T) {
+	diskDetails, err := MockDiskDetails()
```

Comment:
I am trying really hard to understand the test.
Can we try to use table driven tests?
It will be idiomatic & will be very helpful to understand.

I have come to believe/being told that NDM cannot have Table Driven Tests. Is it true?

Reply:
Not exactly ndm @AmitKumarDas there are test cases in controller package, probe package , util package which follows table driven tests.
but for udev package there are some limitation which does not allow us to use table driven tests.

Reply:
In this case tests can be ran in parallel but hard to use td here. I will make some changes and put some comments on that.